### PR TITLE
[DOCS-3880] Add pattern inspector to docs

### PR DESCRIPTION
**Pull request by maycmlee**
### What does this PR do?

Adds info and images about the new pattern inspector feature.

### Motivation

DOCS-3880

---

### Reviewer checklist
- [ ] Review the changed files.
- [ ] Review the URLs listed in the [Preview](#preview) section.
- [ ] Check images for PII
- [ ] Review any mentions of "Contact Datadog support" for internal support documentation.

### CODE REVIEW

File: content/en/logs/explorer/group.md [hunk 1]
```diff
@@ -109,6 +109,21 @@ Patterns support the [List Aggregates][15] visualization. Clicking a pattern in
...
+Use Pattern Inspector to get a visual breakdown of the underlying values of a log pattern’s aggregation based on your search query. For example, if you are investigating an issue, you could see how many hosts are involved or what regions or data centers are impacted.
+
+{{< img src="logs/explorer/group/inspect_values.png" alt="The distribution of values pop-up showing a bar graph of the values" style="width:70%;" >}}
+
+To use Pattern Inspector:
+
+1. Go to [Log Explorer][16].
+2. Click **Patterns** in the **Group into** section. In the list of patterns, the aggregate values in the message section are highlighted in yellow. Hover over an aggregate value to get a preview of the visual distribution of its values. 
+3. Click on an aggregate value to open the log pattern’s side panel. 
+4. In the **Pattern Inspector** tab, click on a value and then **View logs** to see the related logs in the side panel. Click **Search for** to see the related logs in the Log Explorer.
```
Comment (drichards-87) [inline content/en/logs/explorer/group.md:123]:
```suggestion
4. On the **Pattern Inspector** tab, click on a value and select **View logs** to see the related logs in the side panel. Click **Search for** to see the related logs in the Log Explorer.
```

File: content/en/logs/explorer/group.md [hunk 2]
```diff
@@ -109,6 +109,21 @@ Patterns support the [List Aggregates][15] visualization. Clicking a pattern in
 
 {{< img src="logs/explorer/patterns_side_panel.jpg" alt="The log side panel with the view all button and the parsing rule highlighted" style="width:80%;" >}}
 
+### Pattern Inspector
+
+Use Pattern Inspector to get a visual breakdown of the underlying values of a log pattern’s aggregation based on your search query. For example, if you are investigating an issue, you could see how many hosts are involved or what regions or data centers are impacted.
+
+{{< img src="logs/explorer/group/inspect_values.png" alt="The distribution of values pop-up showing a bar graph of the values" style="width:70%;" >}}
```
Comment (drichards-87) [inline content/en/logs/explorer/group.md:116]: In my experience, most style guides recommend not using pop-up as a noun. For example, you can see this reflected in the Microsoft Style Guide and Rackspace Style Guides (both available for free online). That said, it doesn't appear that we have a style guideline for this particular term. And I'm also struggling to come up with a better solution, other than "pop-up window".

**Review by drichards-87 — CHANGES_REQUESTED** [review]

File: content/en/logs/explorer/group.md (hunk 2 above)
Comment (maycmlee) [inline content/en/logs/explorer/group.md:116]:
```suggestion
{{< img src="logs/explorer/group/inspect_values.png" alt="The distribution of values graph showing a bar graph of the values" style="width:70%;" >}}
```

Comment (maycmlee) [reply content/en/logs/explorer/group.md]: Thanks @drichards-87! Ah yeah, we usually don't use pop-up either, I forgot 🙃. I changed it to `graph`. Also applied your other suggestions. Could you re-review?

**Review by drichards-87 — APPROVED** [review]
Changes look good!

**Pull request merged**